src/System.ComponentModel.Primitives/src/System/ComponentModel/Component.cs

```diff
@@ -83,61 +76,68 @@ public void Dispose()
+        ///     Dispose(true);
+        ///     GC.SuppressFinalize(this);
+        /// }
         ///
-        ///    protected virtual void Dispose(bool disposing) {
-        ///        if (disposing) {
-        ///            if (myobject != null) {
-        ///                myobject.Dispose();
-        ///                myobject = null;
-        ///            }
-        ///        }
-        ///        if (myhandle != IntPtr.Zero) {
-        ///            NativeMethods.Release(myhandle);
-        ///            myhandle = IntPtr.Zero;
-        ///        }
-        ///    }
+        /// protected virtual void Dispose(bool disposing)
+        /// {
+        ///     if (disposing)
+        ///     {
+        ///         if (myobject != null)
+        ///         {
+        ///             myobject.Dispose();
+        ///             myobject = null;
+        ///         }
+        ///     }
+        ///     if (myhandle != IntPtr.Zero)
+        ///     {
+        ///         NativeMethods.Release(myhandle);
+        ///         myhandle = IntPtr.Zero;
+        ///     }
+        /// }
         ///
-        ///    ~MyClass() {
-        ///        Dispose(false);
-        ///    }
-        ///    </code>
-        ///    <para>
-        ///    For base classes, you should never override the Finalizer (~Class in C#)
-        ///    or the Dispose method that takes no arguments, rather you should
-        ///    always override the Dispose method that takes a bool. 
-        ///    </para>
-        ///    <code>
-        ///    protected override void Dispose(bool disposing) {
-        ///        if (disposing) {
-        ///            if (myobject != null) {
-        ///                myobject.Dispose();
-        ///                myobject = null;
-        ///            }
-        ///        }
-        ///        if (myhandle != IntPtr.Zero) {
-        ///            NativeMethods.Release(myhandle);
-        ///            myhandle = IntPtr.Zero;
-        ///        }
-        ///        base.Dispose(disposing);
-        ///    }
-        ///    </code>
+        /// ~MyClass()
+        /// {
+        ///     Dispose(false);
+        /// }
+        /// </code>
+        /// 
+        /// For base classes, you should never override the Finalizer (~Class in C#)
+        /// or the Dispose method that takes no arguments, rather you should
+        /// always override the Dispose method that takes a bool. 
+        /// 
+        /// <code>
+        /// protected override void Dispose(bool disposing)
+        /// {
+        ///     if (disposing)
+        ///     {
+        ///         if (myobject != null)
+        ///         {
+        ///             myobject.Dispose();
+        ///             myobject = null;
+        ///         }
+        ///     }
+        ///     if (myhandle != IntPtr.Zero)
+        ///     {
+        ///         NativeMethods.Release(myhandle);
+        ///         myhandle = IntPtr.Zero;
+        ///     }
```

Comment:
This code doesn't need to document the Dispose pattern :)  Everything except for the first summary line can be deleted.